modules/claims_api/spec/sidekiq/report_hourly_unsuccessful_submissions_spec.rb

```diff
@@ -104,25 +104,26 @@
                                                                                  transaction_id: 'transaction_3',
                                                                                  id: '4')
 
-        FactoryBot.create(:auto_established_claim_va_gov,
-                          :errored,
-                          created_at: 30.seconds.ago,
-                          evss_response: "[{'status'=>'422', " \
-                                         "'title'=>'Backend Service Exception', " \
-                                         "'detail'=>'The Maximum number of EP codes have been " \
-                                         "reached for this benefit type claim code'}]",
-                          transaction_id: 'transaction_4')
-
-        FactoryBot.create(:auto_established_claim_va_gov,
-                          :errored,
-                          created_at: 120.seconds.ago,
-                          evss_response: "[{'status'=>'422', " \
-                                         "'title'=>'Backend Service Exception', " \
-                                         "'detail'=>'Claim could not be established. Retries will fail.'}]",
-                          transaction_id: 'transaction_5')
+        claim_five = FactoryBot.create(:auto_established_claim_va_gov,
+                                       :errored,
+                                       created_at: 30.seconds.ago,
+                                       evss_response: [{ 'status' => '422',
+                                                         'title' => 'Backend Service Exception',
+                                                         'detail' => 'The Maximum number of EP codes have been ' \
+                                                                     'reached for this benefit type claim code' }],
+                                       transaction_id: 'transaction_4')
+
+        claim_six = FactoryBot.create(:auto_established_claim_va_gov,
+                                      :errored,
+                                      created_at: 120.seconds.ago,
+                                      evss_response: [{ 'status' => '422',
+                                                        'title' => 'Backend Service Exception',
+                                                        'detail' => 'Claim could not be established. ' \
+                                                                    'Retries will fail.' }],
+                                      transaction_id: 'transaction_5')
```

Comment:
These are saved as data type Array on the record, not a string, I fooled myself when printing out to the console whle grabbing them.